pkg/volume/flocker/client.go

```diff
@@ -0,0 +1,382 @@
+/*
```

Comment:
It was before, it isn't now, so move it out is going to be a good decision. Doing it just now.